portfolio/src/test/java/com/google/sps/servlets/DeleteDataServletTest.java

```diff
@@ -30,19 +32,33 @@ public void setUp() {
   }
 
   @Test
-  public void testDeleteComments() {
+  public void testDeleteCommentsSuccess() {
```

Comment:
Can you also test a fail test case?

Reply:
Hmm, probably no, I don't know what is "fail" here. I added it here just to indicate that this function tests that comments are deleted successfully. I mean, I can measure success here, which is "Comments that are expected to be deleted are deleted indeed".

Reply:
well, "fail" would be that comments that are expected to be deleted are not deleted, I wanted to say that I don't know how to fail that in test

Reply:
I am really worried about how there is a POST method that doesn't check for Authentication and just delete stuff. Will the method be available to anything other than client? (i.e would someone sending a HTTP request using curl work?)

Reply:
Oh, my bad. I will add auth check on backend

Reply:
I've added the fail test cases in the PR#20, can I mark this conversation as resolved?